cmd/dep/ensure.go

```diff
@@ -292,14 +295,7 @@ func (cmd *ensureCommand) runVendorOnly(ctx *dep.Ctx, args []string, p *dep.Proj
 	}
 
 	if cmd.dryRun {
-		ctx.Out.Printf("Would have populated vendor/ directory from %s", dep.LockName)
-		if ctx.Verbose {
-			err := sw.PrintPreparedActions(ctx.Err)
-			if err != nil {
-				return errors.WithMessage(err, "prepared actions")
-			}
-		}
-		return nil
+		return sw.PrintPreparedActions(ctx.Err, ctx.Verbose)
```

Comment:
I see that now in some other places, but maybe we need an issue to document the policy and audit/align the existing usages.